src/main/java/com/ssafy/ssafybox/crud/model/service/CRUDServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.ssafy.ssafybox.crud.model.service;
+
+import com.ssafy.ssafybox.crud.model.dto.TodoDto;
+import com.ssafy.ssafybox.crud.model.mapper.CRUDMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CRUDServiceImpl implements CRUDService {
+
+    private final CRUDMapper CRUDMapper;
+
+    @Autowired
+    public CRUDServiceImpl(CRUDMapper CRUDMapper) {
+        this.CRUDMapper = CRUDMapper;
+    }
+
+    @Override
+    public boolean createTodo(String content) {
+        try {
+            CRUDMapper.createTodo(content);
+            return true; // 성공적으로 추가됨
+        } catch (Exception e) {
+            return false; // 예외 발생, 실패
+        }
+    }
+
+    @Override
+    public List<TodoDto> readAllTodo() {
+        return CRUDMapper.readAllTodo();
+    }
+
+    @Override
+    public boolean updateTodo(Long todoId) {
+        try {
+            CRUDMapper.updateTodo(todoId);
+            return true; // 성공적으로 추가됨
+        } catch (Exception e) {
+            return false; // 예외 발생, 실패
+        }
+    }
+
+    @Override
+    public boolean deleteTodo(Long todoId) {
+        try {
+            CRUDMapper.deleteTodo(todoId);
+            return true; // 성공적으로 추가됨
+        } catch (Exception e) {
+            return false; // 예외 발생, 실패
```

Comment:
성공이든 실패든 우리가 알 수 있는 방법이 응답결과 뿐인데 수업때 배운 @Sl4J를 쓴다면 쉽게 확인이 가능해요
```suggestion
log.debug("삭제 실패: {}, e)
return false; // 예외 발생, 실패
```

Reply:
```suggestion
            log.error(" deletion failed: {}, e.getMessage());
            return false;
```
이렇게 명확하게 사용하는 것을 추천 드립니다!